app/models.py

```diff
@@ -480,11 +480,7 @@ class Service(db.Model, Versioned):
     go_live_at = db.Column(db.DateTime, nullable=True)
 
     organisation_id = db.Column(UUID(as_uuid=True), db.ForeignKey('organisation.id'), index=True, nullable=True)
-    organisation = db.relationship(
-        'Organisation',
-        secondary=organisation_to_service,
-        uselist=False,
-        single_parent=True)
+    organisation = db.relationship('Organisation', foreign_keys=[organisation_id])
```

Comment:
If we stick `backref='services'` here we don't need to define the relationship on the Organisation model above

also the foreign_keys field is only needed when there's multiple columns with the same foreign key (eg go_live_user_id vs created_by_id)